python/lsst/atmospec/spectraction.py

```diff
@@ -157,6 +178,15 @@ def spectractorImageFromLsstExposure(self, exp, *, target_label='', disperser_la
         image.hour_angle = asCoords.Angle(ha, unit="deg")
 
         image.data = self._getImageData(exp)
+
+        def _translateCentroid(dmXpos, dmYpos):
+            newX = dmYpos
+            newY = dmXpos
+            return newX, newY
+        image.target_guess = _translateCentroid(xpos, ypos)
```

Comment:
Am I correct that this function is anticipating a fixed offset that will need to be applied?  Otherwise I don't see what it's doing.

Reply:
No - it used to apply tweaks depending on whether the image was being transposed, but I think we're moving away from that ever not being done now, so it's likely now a vestigial function. If you're OK with it, I'll merge it as-is for now, after adding a code comment that it can likely be removed in future, like when I do a big cleanup.